operations/governance-remediation-process.md

```diff
@@ -0,0 +1,83 @@
+
+In rare cases, governance failure may be accompanied by possible CoC violations by individual project leaders. Should this happen, the assigned team will engage the CNCF Code of Conduct Committee and the two teams will coordinate their evaluation.
+
+### Out of Scope:
+
+**Conduct Issues**: Handled by the CNCF Code of Conduct [processes](https://github.com/cncf/foundation/blob/main/code-of-conduct.md#reporting). This includes behavior by project leaders which is a violation of the CNCF CoC; such behavior should be reported to the CoCC and not handled through Governance Remediation.
+
+**Technical Disagreements**: Projects should use or establish governance to resolve such disputes independently.
+
+**Lifecycle Processes**: Elements already covered in the project's lifecycle review.
+
+## Reporting Governance Remediation
+
+**Public Reports: **In most cases, project governance issues will be reported publicly to the TOC by filing [an Issue in the TOC repository](https://github.com/cncf/toc/issues) of type Governance. Should a public issue be reported in some other venue (the project's issue tracker, mailing list, etc.) an issue in the TOC repo should be created for tracking and assignment.
+
+**Anonymous Reports**: Should the reporter not wish their name to be known, they should email &lt;TOC-confidential-email-here>.  The TOC team will then create an issue in the TOC repo tracking the remediation. The identity of the anonymous reporter will be known only to the TOC team.
+
+**Confidential Reports**: Should there be risks around the details of the remediation request being public, the reporter will email &lt;TOC-confidential-email-here>.  The TOC and the assessment team will then use privately shared documents to track the status of the remediation request.
+
+## Governance Remediation Process
+
+### Roles
+
+* TAG Contributor Strategy: For missing governance the assignee will lead, consulting and informing with the assigned TOC member as needed. For non-practiced governance, they act as an advisor to the TOC
+* TOC member assignee(s): serves an oversight/informational function that the governance assignee can leverage as needed to impress upon the project the needed changes or leads the discussion with the project in the case of non-practiced governance
+* Code of Conduct Committee: if a failure of governance coincides with CoC violations by one or more project leaders, the CoCC will coordinate with the assessment team.
+
+### Missing Governance
```

Comment:
As written currently, this part seems _super vague_ and open to arbitrarily overriding project governance.

Reply:
> It is checked when they join and go through all the levels. 

Then "non-practiced governance" remediation should be sufficient?

> There are situations that can happen in between those checkpoints though.

Such as?

Reply:
Example: Etcd.

There was no process in Etcd for removing inactive maintainers, and the fact that inactive maintainers had not been removed meant that quorum to vote on removal couldn't be obtained.  As such, the project needed TOC support to implement a maintainer list overhaul outside of existing governance processes -- and add a removal-for-inactivity paragraph.

Reply:
> There was no process in Etcd for removing inactive maintainers, and the fact that inactive maintainers had not been removed meant that quorum to vote on removal couldn't be obtained. As such, the project needed TOC support to implement a maintainer list overhaul outside of existing governance processes -- and add a removal-for-inactivity paragraph.

Inability to achieve quorum seems like another variation on "non-practiced governance", which is a much narrower exception to permit overriding project governance, rather than a general purpose governance override.

Reply:
Ben: it feels like you're making a semantic argument.  What outcome are you hoping for, here?

Reply:
@BenTheElder I think Liz's wording makes it a bit more clear [here](https://github.com/cncf/toc/pull/1417/commits/e9acf782bdc1cdb761ac75946c64e5b8aa4c13da). Does that answer your question?

Reply:
> Ben: it feels like you're making a semantic argument. What outcome are you hoping for, here?

1) Semantics matter?
2) I don't think it's just semantic.

I think we should effectively cover: "the project is no longer able to govern as previously agreed" -> there is an intervention process in that case (and any variation on it).

The initial semantic leaves a gaping, under-defined "missing governance" case with no examples, that could easily be interpreted to be _any_ governance choice not matching the TOC's desires.

---

Thanks @xmulligan @lizrice  I agree that https://github.com/cncf/toc/pull/1417/commits/e9acf782bdc1cdb761ac75946c64e5b8aa4c13da is much clearer and that it makes sense to handle this case 🙏

Reply:
Where I said a semantic arguement is because you asked me what missing governance was, I gave you an example, and then you argued that my example wasn't missing governance.

Agreed that Liz's definition of missing governance is def better than the original draft and we should adopt it.

Reply:
Great, I'm going to resolve this conversation then since I've added the modified wording